File: src/main/java/cz/kamenitxan/jakon/core/function/IFuncion.java
package cz.kamenitxan.jakon.core.function;

import java.util.Map;

/**
 * Created by TPa on 25.05.16.
 */
public interface IFuncion {
	default String getName() {
		return this.getClass().getSimpleName();
	}

	String execute(Map<String, String> params);
}

File: src/main/java/cz/kamenitxan/jakon/core/function/IFuncion.scala
package cz.kamenitxan.jakon.core.function

import java.util

/**
  * Funkce jsou zapisovane v tvaru {jmenoFunkce nazevParametru=hodnotaparametru...}
  * Created by TPa on 25.05.16.
  */
trait IFuncion {
	def getName: String = {
		this.getClass.getSimpleName
	}

	def execute(params: util.Map[String, String]): String
}